app/controllers/concerns/api_responders.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ApiResponders
+  extend ActiveSupport::Concern
+
+  private
+
+    def respond_with_error(message, status = :unprocessable_entity, context = {})
+      is_exception = message.kind_of?(StandardError)
```

Comment:
This is wrong. We shouldnt be handling error kinds here. Thats why we have error rescuers in the first place. This should be something like
```
def respond_with_error(message:, status: :internal_server_error)
   render status: status, json: { error: message }
end
```

Reply:
Actually, we are passing exceptions instead of messages.

Reply:
Just pass the message.